.github/workflows/main.yml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Sorry, I didn't realise the syntax. Looking at the [setup-python](https://github.com/actions/setup-python) README, it looks like the version numbers should be in quotes:
```suggestion
        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
```